install/kubernetes/helm/istio/values.yaml

```diff
@@ -383,6 +383,7 @@ global:
     amd64: 2
     s390x: 2
     ppc64le: 2
+    arm64: 2
```

Comment:
Won't we have  an amd64 image and an arm64 image? So now our amd64 images may be scheduled on arm64 nodes?

Reply:
yes, there is no arm64 image. So i do some modification for supporting build arm64 images.